Neos.Flow/Classes/SignalSlot/Dispatcher.php

```diff
@@ -149,7 +149,7 @@ public function dispatch(string $signalClassName, string $signalName, array $sig
         }
 
         foreach ($this->slots[$signalClassName][$signalName] as $slotInformation) {
-            $finalSignalArguments = $signalArguments;
+            $finalSignalArguments = array_values($signalArguments);
```

Comment:
Not sure if that's the proper fix, but I got an error
> named argument "$response" doesn't exist

(or similar) because PHP 8 tries to map this to a named argument if the array contains associative keys

Reply:
FYI: WIth PHP 8.1 this won't be required any longer (https://wiki.php.net/rfc/array_unpacking_string_keys)

Reply:
Yeah, but will be some time until we have PHP >= 8.1 minimum :)